spec/models/sushi_spec.rb

```diff
@@ -142,4 +142,54 @@
       it { is_expected.to be_nil }
     end
   end
+
+  describe "YearOfPublicationCoercion" do
+    subject(:instance) { klass.new(params) }
+
+    let(:klass) do
+      Class.new do
+        include Sushi::YearOfPublicationCoercion
+        def initialize(params = {})
+          coerce_yop(params)
+        end
+      end
+    end
```

Comment:
I tried to write up a spec for `QueryParameterValidation` in here last week but couldn't get it to work. I think something like this is what I was missing.